### DISCUSSION TITLE
Swizzin Dashboard  - Network Info error

### DISCUSSION THREAD

**Discussion by infoextreme** [swizzin/swizzin#1176] [General] [answered]
Hello, Swizzin is working normally. 
Except, Dashboard the Network Info isn't being updated. 
How can do I reset or solve this problem?

![image](https://github.com/user-attachments/assets/fb614942-0649-4397-93fc-3f098429e02b)

Last Update was Oct, 2023.

Thanks.

**Comment by liaralabs** [comment]
The visibility of stats for hourly/daily/weekly indicates that everything is working as expected. 

The bottom view are your top10 days, sorted by total bandwidth (up+down). This indicates you have not beaten any of these days in several months.

**Reply by infoextreme** [reply]
There is daily traffic, just like yesterday and today but it is not being registered.

**Reply by liaralabs** [reply, marked as answer]
The backend for the statistics is `vnstat` not `ifconfig`. That said, the screen you shared doesn't seem to indicate any issues with vnstat recording the metrics

**Reply by infoextreme** [reply]
Thanks, the problem was with vnstat service. 
Now is working normally.

**Comment by JSouthGB** [comment]
For anyone else that comes across this in the future. Check that the vnstat systemd service is running.

`systemctl status vnstat.service`

If it isn't:
`sudo systemctl start vnstat.service`
